services-camel/src/test/resources/destroy-derivatives-router-it-context.xml

```diff
@@ -0,0 +1,95 @@
+        http://www.springframework.org/schema/util/spring-util.xsd
+        http://www.springframework.org/schema/context
+        http://www.springframework.org/schema/context/spring-context-4.3.xsd
+        http://camel.apache.org/schema/spring
+        http://camel.apache.org/schema/spring/camel-spring.xsd">
+
+    <bean id="properties" class="org.apache.camel.component.properties.PropertiesComponent">
+        <property name="location" value="classpath:destroy-derivatives-it-config.properties"/>
+    </bean>
+
+    <bean id="bridgePropertyPlaceholder" class="org.apache.camel.spring.spi.BridgePropertyPlaceholderConfigurer">
+        <property name="location" value="classpath:destroy-derivatives-router-it-config.properties"/>
+    </bean>
+
+    <bean id="binaryInfoProcessor" class="edu.unc.lib.dl.services.camel.destroyDerivatives.BinaryInfoProcessor">
+        <constructor-arg value="${sourceImages.dir}" />
+    </bean>
+
+    <bean id="destroyCollectionSrcImgProcessor" class="org.mockito.Mockito" factory-method="mock">
+        <constructor-arg value="edu.unc.lib.dl.services.camel.destroyDerivatives.DestroyDerivativesProcessor" />
+    </bean>
+
+    <bean id="destroySmallThumbnailProcessor" class="org.mockito.Mockito" factory-method="mock">
+        <constructor-arg value="edu.unc.lib.dl.services.camel.destroyDerivatives.DestroyDerivativesProcessor" />
+    </bean>
+
+    <bean id="destroyLargeThumbnailProcessor" class="org.mockito.Mockito" factory-method="mock">
+        <constructor-arg value="edu.unc.lib.dl.services.camel.destroyDerivatives.DestroyDerivativesProcessor" />
+    </bean>
+
+    <bean id="destroyAccessCopyProcessor" class="org.mockito.Mockito" factory-method="mock">
+        <constructor-arg value="edu.unc.lib.dl.services.camel.destroyDerivatives.DestroyDerivativesProcessor" />
+    </bean>
+
+    <bean id="destroyFulltextProcessor" class="org.mockito.Mockito" factory-method="mock">
+        <constructor-arg value="edu.unc.lib.dl.services.camel.destroyDerivatives.DestroyDerivativesProcessor" />
+    </bean>
+
+    <bean id="binaryDestroyedJmsTemplate" class="org.springframework.jms.core.JmsTemplate">
+        <property name="connectionFactory" ref="jmsFactory" />
+        <property name="defaultDestinationName" value="${cdr.destroy.derivatives.stream}" />
+        <property name="pubSubDomain" value="false" />
+    </bean>
+
+    <bean id="binaryDestroyedMessageSender" class="edu.unc.lib.dl.services.MessageSender">
+        <property name="jmsTemplate" ref="binaryDestroyedJmsTemplate" />
+    </bean>
+
+    <bean id="contentPathFactory" class="edu.unc.lib.dl.fedora.ContentPathFactory"
+          init-method="init">
+        <property name="cacheMaxSize" value="100" />
+        <property name="cacheTimeToLive" value="5000" />
+        <property name="fcrepoClient" ref="fcrepoClient" />
+    </bean>
+
+    <bean id="objectAclFactory" class="edu.unc.lib.dl.acl.fcrepo4.ObjectAclFactory"
```

Comment:
I think you can remove `objectAclFactory` and `inheritedPermissionEvaluator` since they are already being initialized in `acl-service-context.xml`